libs/books/data-access/src/lib/+state/reading-list.effects.spec.ts

```diff
@@ -42,4 +43,47 @@ describe('ToReadEffects', () => {
       httpMock.expectOne('/api/reading-list').flush([]);
     });
   });
+
+  it('should finish reading the book', done => {
+    const item: ReadingListItem = createReadingListItem('A');
+    actions = new ReplaySubject();
+    actions.next(ReadingListActions.markedAsFinishedReading({item}));
+
+    const finishedBook = {
+      ...item,
+      finished: true,
+      finishedDate: new Date().toISOString()
+    };
+
+    effects.markBookAsFinished$.subscribe(action => {
+      expect(action).toEqual(ReadingListActions.confirmedMarkedAsFinishedReading({ item: finishedBook}));
+      done();
+    });
+
+    httpMock.expectOne({
+      url: `/api/reading-list/${item.bookId}/finished`,
+      method: 'put'
+    }).flush(finishedBook);
+  })
+
+  it('should throw finish reading book api error', done => {
```

Comment:
Provide description for all the assertions in proper format.
Format : should [do this] when [this happens]